src/js/themes/darkTheme/colors.ts

```diff
@@ -54,31 +56,37 @@ export const colors: ThemeColors = {
   },
   error: {
     default: '#D73847',
-    alternative: '#E06470',
+    defaultHover: '#FF4D58',
+    defaultPressed: '#F9868E',
```

Comment:
Matches figma and has css in js and css variable token

![Screenshot 2024-01-18 at 6 00 11 PM](https://github.com/MetaMask/design-tokens/assets/8112138/20fb9dfd-cc26-4b41-9b45-a2d356de69b8)